DragaliaAPI/Models/Data/DewValues.cs

```diff
@@ -4,17 +4,17 @@ namespace DragaliaAPI.Models.Data;
 
 public enum CharaPromoteValues
 {
-    TO_RARITY_4 = 2500,
-    TO_RARITY_5 = 25000
+    ToRarity4 = 2500,
+    ToRarity5 = 25000
 }
 
 public enum DupeReturnBaseValues
 {
-    RARITY_4_STORY = 100,
-    RARITY_3 = 150,
-    RARITY_5_STORY = 300,
-    RARITY_4 = 2200,
-    RARITY_5 = 8500
+    Rarity4Story = 100,
+    Rarity3 = 150,
+    Rarity5Story = 300,
+    Rarity4 = 2200,
+    Rarity5 = 8500
 }
```

Comment:
Should this and the other eldwater values not be constants? As far as I can tell you just immediately put them into a Dictionary<int, int> so there is no value in having a type for it